### PR TITLE
Added Team Member Ayush's info :D

### PR DESCRIPTION
**Pull request by yashk0thari**
Added image of Team Member :D And Info too :D

### CODE REVIEW

**Comment by yashk0thari** [comment]
https://beta-king.github.io/aragogo.fossasia.org/team.html --> Preview Link :D

File: team.html [hunk 1]
```diff
@@ -1396,6 +1396,23 @@ <h1>Developers</h1>
...
+								</div>
+							</div>
+						</div>
+						<span class="speaker-name">Ayush Gupta</span>
+						<span>Developer</span>
+					</div>
+				</div>
+
+						</div>
+				</div>
```
Comment (sayak119) [inline team.html]: Extra div tag and please indent properly

**Review by sayak119 — CHANGES_REQUESTED** [review]

**Comment by yashk0thari** [comment]
Indented :D

File: team.html [hunk 2]
```diff
@@ -1396,6 +1396,23 @@ <h1>Developers</h1>
...
+									<a href="https://github.com/agbilotia1998"><i class="icon fa fa-github"></i></a>
+								</div>
+							</div>
+						</div>
+						<span class="speaker-name">Ayush Gupta</span>
+						<span>Developer</span>
+					</div>
+				</div>
+
+				</div>
```
Comment (sayak119) [inline team.html]: @Beta-King this div tag is extra

**Review by sayak119 — CHANGES_REQUESTED** [review]

File: team.html (hunk 2 above)
Comment (yashk0thari) [reply team.html]: all three ??

Comment (harshit98) [reply team.html]: Remove div tag from line 1414. It is extra.

Comment (yashk0thari) [reply team.html]: Ohk :D

**Comment by yashk0thari** [comment]
Made changes :D thanks for the help :D

**Comment by yashk0thari** [comment]
Squashed :D

**Comment by yashk0thari** [comment]
Can we merge now ??

**Review by akshatnitd — APPROVED** [review]
LGTM :rocket:

**Pull request merged**